packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -460,6 +483,9 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
   void acceptGesture(int pointer) {
     if (_state == _ScaleState.possible) {
       _state = _ScaleState.started;
+      if (dragStartBehavior == DragStartBehavior.start) {
```

Comment:
Not too familiar with the scale gesture recognizer: why this over calling `_reconfigure`?

Reply:
I'm not either but I think you're right. I also don't need to set _initialSpan at the other location above because reconfigure is called just before it.  Seems to work out cleaner and give me more confidence 👍

Reply:
Nevermind, that broke a bunch of tests.  Looks like I can't reset those values here without breaking the pan gesture.  I've kept this one the same but I was still able to remove the other set of _initialSpan, so it's a bit cleaner.

Reply:
`_reconfigure` calls the `onEnd` callback, that's probably undesirable in our case. But I think we want to reset other states here too:
```
    _initialFocalPoint = _currentFocalPoint;
    _initialSpan = _currentSpan;
    _initialLine = _currentLine;
    _initialHorizontalSpan = _currentHorizontalSpan;
    _initialVerticalSpan = _currentVerticalSpan;
```

Reply:
Ah good call. Fixed and ready for re-review.